### PR TITLE
Directory size plugin

### PR DESCRIPTION
**Pull request by dajobe**

### CODE REVIEW

File: directory.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+DIR=$1
+
+size=-1
+if [ ! -d $DIR ]; then
+    status="err No such directory $DIR"
+else
+    status="ok directory $DIR found"
+    size=`du -sm $DIR 2>/dev/null | awk '{print $1}'`
+    if [ "X$size" = "X" ]; then
```
Comment (robert-chiniquy) [inline directory.sh]: could be `[ -z "$size" ]` instead of the `"X"` hack

Comment (dajobe) [inline directory.sh]: I remember getting burned by the portability of this to something (solaris, AIX, ksh, ...?) a few times.  No absolute reference but http://unix.stackexchange.com/questions/32180/testing-if-a-variable-is-empty-in-a-shell-script points to a warning at http://resources.mpi-inf.mpg.de/departments/rg1/teaching/unixffb-ss98/quoting-guide.html#ssect:appl-test

**Comment by mburns** [comment]
LGTM :+1:

**Comment by rphillips** [comment]
+1

**Pull request merged**